config/samsung-heatqlte.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# DTB: From AP_G357FZXXS1AQA2_CL572060_QB12156321_REV00_user_low_ship_MULTI_CERT.tar
+
+OPTIONS=(-r vddio -r vdd --backlight-fallback-dcs --dcs-no-get-brightness)
```

Comment:
The DCS backlight control (sadly) probably doesn't work on Samsung AMOLED panels, so this should probably be `-r vddio -r vdd --no-backlight`.

Reply:
Thanks. I've updated this.